2019-05-14-armagaddon/heka/reprocess-structured-landfill/script.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+# NOTE, need to patch
+# https://github.com/mozilla-services/lua_sandbox_extensions/blob/master/moz_telemetry/io_modules/moz_telemetry/s3.lua#L169
+# before running this because there's a "-" in the "host" partition for structured ingestion
+set -e
+
+# We need to run a hindsight job for each day (instead of a single hindsight
+# job for all days) because the s3 output only supports writing partitions
+# based on fields, and submission date (derived from Timestamp) isn't a field
+# for raw messages.
+for date in $(seq 20190504 20190511) ; do
+  sed "s/FIXME/$date/" landfill.json.tmp > landfill.json
+  sed "s/FIXME/$date/" hindsight/run/output/telemetry_s3.cfg.tmp > hindsight/run/output/telemetry_s3.cfg
+  cd hindsight
+  hindsight_cli hindsight.cfg 7
+  cd ..
+  rm -rf /media/ephemeral0/hindsight_output
+  rm -f hindsight/run/input/landfill*cfg
```

Comment:
The tmp directory could also be wiped but it shouldn't cause any problems if it isn't